### PR TITLE
Blackbox Hub

### PR DESCRIPTION
**Pull request by CaitPrough**
## Submission Checklist:

- [ x] I am a current high school, middle school, or home schooled student.
  - [ x] I have filled out the [verification form](https://airtable.com/app4Bs8Tjwvk5qcD4/pagxECjJZOgvKVnLd/form)
- [ x] I have joined the [`#electronics`](https://hackclub.slack.com/archives/C056AMWSFKJ) channel on [Slack](https://hackclub.com/slack/?event=onboard)
- [ x] I made this board from scratch, even if I followed a tutorial
- [ x] I have followed the directions in [README.md](https://github.com/hackclub/OnBoard/blob/main/README.md#getting-started)
  - [ x] Created a folder under `onboard/projects`
  - [ x] Filled out `TEMPLATE.md` as `README.md` inside your project folder
  - [ x] Uploaded `gerber.zip` and `schematic.pdf` inside your project folder
  - [ x] Created a folder called `src` and uploaded design files
  - [ x] Uploaded screenshot of PCB order called `cart.png` with all costs included
  - [ x] I am ordering the smallest batch size (ie. JLCPCB's smallest order is for 5 boards)
  - [ ] If outside the US I've checked that I can afford the customs charges in my country (which isn't covered by the grant)

- [ x] (Optional) This project is from a tutorial[^1].
- [ x] (Optional) This project is for [the Trail](https://trail.hackclub.com/).
- [ x] (Optional) I'm in a FIRST (FRC, FTC, FLL, etc.) team. The number is: 8825

[^1]: Projects from a tutorial are 100% fine! We just want to ask so we can count how people are using tutorials.

*Tutorial was used as a base. This was not a project for First but I am in a team

### CODE REVIEW

**Comment by prophetorpheus** [comment]
Hi, I'm Orpheus Leap! Here to help you review your PR.
  
**projects/BLACKBOX_HUB:**

Required files
||||
|---|---|---|
|✅| README.md | A description of your project |
|✅ | cart.png |  |
|✅| gerber.zip | |
|✅ | [schematic.pdf](<https://raw.githubusercontent.com/hackclub/OnBoard/566c25a599a6a15fc14d79b67305e3a69a6e575a/projects/BLACKBOX_HUB/schematic.pdf>) | Manually check [schematic.pdf](<https://raw.githubusercontent.com/hackclub/OnBoard/566c25a599a6a15fc14d79b67305e3a69a6e575a/projects/BLACKBOX_HUB/schematic.pdf>) |
|✅| Source files - EasyEDA |  Found both PCB and SCH files |

You can view a render of your board over on [gerber.zip/2d](<https://gerber.zip/2d/?mode=layers&boardUrl=https://raw.githubusercontent.com/hackclub/OnBoard/566c25a599a6a15fc14d79b67305e3a69a6e575a/projects/BLACKBOX_HUB/gerber.zip>)!
		
  Happy OnBoarding!
 <sub>This bot is a simpler helper for common submission types. If there's an error with this, please point it out and someone from the OnBoard team will look at it manually.</sub>

**Comment by 24c02** [comment]
Screenshot 2024-05-28 at 4 23 57 PM
hey, i'm not sure these 2 USB-C upstream ports would work – if they did you'd need to connect them to separate data lines, i wouldn't have them overlapping like that (very jarring to look at on a schematic, hard to follow), and wouldn't it make more sense to have 1?

**Comment by 24c02** [comment]
other than that, this seems to check out electrically! nice work! :-)

**Comment by 24c02** [comment]
Screenshot 2024-05-28 at 4 23 57 PM
what's going on here?

**Comment by TheTridentGuy** [comment]
That should be two usbs wired the same, but now that i see the no connect flags, ill check its wired right.

**Comment by 24c02** [comment]
please don't have those symbols on top of each other, it's difficult to see what's going on...
also, i think that's the input side of the hub? unless that's a male USB port it's not going to work
additionally shorting 2 USB ports worth of data lines won't work, that's what a hub is for

**Comment by TheTridentGuy** [comment]
Yeah, i know its super confusing but i didnt know another way to do it. Only one of the usbs will be used at a time, and really im only using the holes for them. Theres two because i designed it to work with two different models of raspberry pi.

**Comment by 24c02** [comment]
are they male USBs though? otherwise there's no way that plug works :-/

**Comment by TheTridentGuy** [comment]
They arent even getting placed, theyre just holes that either the raspberrypis pins will fit into, or ill have to solder jumpers to. Im aware that this will render one of the usbs on the pi unusable.

**Comment by 24c02** [comment]
OH! okay, that makes a lot more sense, sorry...
that area of the schematic is a crime, but it should work!
[grant](https://hcb.hackclub.com/grants/q16IaG) granted :-)

**Comment by TheTridentGuy** [comment]
Btw, is there a way i could do that better in easyeda? And thanks for reviewing :)
